[2024-12-04T15:27:26.276+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T15:27:05.002274+00:00 [queued]>
[2024-12-04T15:27:26.306+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T15:27:05.002274+00:00 [queued]>
[2024-12-04T15:27:26.325+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-12-04T15:27:26.672+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_merge_FO> on 2024-12-04 15:27:05.002274+00:00
[2024-12-04T15:27:26.719+0000] {standard_task_runner.py:57} INFO - Started process 882 to run task
[2024-12-04T15:27:26.757+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'FoReport_file_processing', 'etl_merge_FO', 'manual__2024-12-04T15:27:05.002274+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/FoReport_file_processing.py', '--cfg-path', '/tmp/tmpncpb2bim']
[2024-12-04T15:27:26.765+0000] {standard_task_runner.py:85} INFO - Job 114: Subtask etl_merge_FO
[2024-12-04T15:27:27.288+0000] {task_command.py:410} INFO - Running <TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T15:27:05.002274+00:00 [running]> on host effe9442f8f2
[2024-12-04T15:27:27.535+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='FoReport_file_processing' AIRFLOW_CTX_TASK_ID='etl_merge_FO' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T15:27:05.002274+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-04T15:27:05.002274+00:00'
[2024-12-04T15:27:32.048+0000] {logging_mixin.py:150} INFO - No data to merge.
[2024-12-04T15:27:32.140+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-04T15:27:32.305+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=FoReport_file_processing, task_id=etl_merge_FO, execution_date=20241204T152705, start_date=20241204T152726, end_date=20241204T152732
[2024-12-04T15:27:32.384+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-04T15:27:32.429+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
